app/models/project.rb

```diff
@@ -64,6 +67,18 @@ class Project < ActiveRecord::Base
 
   alias_method :logs, :project_logs
 
+  # Configure how the class is duplicated
+  amoeba do
+    enable
+    propagate
+
+    # The default name is computed, if it hasn't been set it will be blank.
+    # We need to manually copy over the name and set it here for it to work.
+    customize(lambda { |original, new|
+      new.name = "Copy of #{original.display_name}"
```

Comment:
Maybe choose a different var name here instead of `original` since that is also used as an association? Maybe just `orig`?